packages/vscode-extension/.vscode/launch.json

```diff
@@ -22,7 +22,6 @@
       "type": "extensionHost",
       "request": "launch",
       "args": [
-        "--disable-extensions",
```

Comment:
It's kind of tricky since our extension depends on Azure extension and our unit tests on extension somehow depends on vscode context.